src/PowerSimulations.jl

```diff
@@ -128,43 +129,50 @@ export run_economic_dispatch
 export run_unit_commitment
 export set_device_model!
 export set_service_model!
-export set_transmission_model!
-export get_transmission_model
+export set_network_model!
+export get_network_formulation
 ## Results interfaces
 export SimulationResultsExport
 export ProblemResultsExport
 export export_results
-export get_existing_duals
-export get_existing_parameters
-export get_existing_timestamps
-export get_existing_variables
-export get_problem_name
+export get_dual_values
+export get_parameter_values
+export get_variable_values
+export get_timestamps
+export get_model_name
```

Comment:
can we also export `get_jump_model`

Reply:
The interface should be on `get_optimization_container`